Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -60,116 +66,80 @@ Steps to perform (extension developer)
-       \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPiFlexFormValue(
-           $pluginSignature,
-           // FlexForm configuration schema file
-           'FILE:EXT:example/Configuration/FlexForms/Registration.xml'
-       );
-
-   .. tip::
-
-      The plugin signature is used in the database field `tt_content.list_type`
-      as well, when the tt_content record is saved. If you are confused about
-      how to handle underscores and upper / lowercase, check there to see
-      what your plugin signature is.
-
-   .. versionadded:: 12.0
-      The method :php:`ExtensionUtility::registerPlugin()` returns the plugin signature.
-
-   Also look on the page :ref:`extension-naming`.
-
-   If you are using a content element instead of a plugin, the example
-   will look like this:
-
-   .. code-block:: php
-      :caption: Add in file EXT:your_extension/Configuration/TCA/Overrides/tt_content.php
-
-      \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPiFlexFormValue(
-         // 'list_type' does not apply here
-         '*',
-         // FlexForm configuration schema file
-         'FILE:EXT:example/Configuration/FlexForms/Registration.xml',
-         // ctype
-         'accordion'
-      );
-
-   Finally, according to "Configuration of the displayed order of fields in FormEngine
-   and their tab alignment." the field containing the FlexForm still needs to be
-   added to the `showitem` directive.
-   The following example shows line from the accordion element of the Bootstrap Package.
-
-   .. code-block:: php
-      :caption: Add in file EXT:your_extension/Configuration/TCA/Overrides/tt_content.php
-      :emphasize-lines: 11
-
-      // Configure element type
-      $GLOBALS['TCA']['tt_content']['types']['accordion'] = array_replace_recursive(
-         $GLOBALS['TCA']['tt_content']['types']['accordion'],
-         [
-            'showitem' => '
-               --div--;General,
-               --palette--;General;general,
-               --palette--;Headers;headers,
-               tx_bootstrappackage_accordion_item,
-               --div--;Options,
-               pi_flexform'
-         ]
-      );
+#.  Create configuration schema in :ref:`T3DataStructure <t3ds>` format (XML)
+
+    **Example:**
+
+    ..  include:: /CodeSnippets/FlexForms/Examples/PluginHaikuList.rst.txt
+
+
+#.  The configuration schema is attached to one or more plugins in the folder
+    :file:`Configuration/TCA/Overrides` of an extension.
+
+    Example for the FlexForm registration of a basic plugin:
+
+    ..  include:: /CodeSnippets/FlexForms/Examples/PluginHaikuListRegistration.rst.txt
+
+    ..  versionadded:: 12.0
+        The method :php:`ExtensionUtility::registerPlugin()` returns the plugin signature.
+
+    When registering Extbase plugins you can use the return value of
+    :php:`ExtensionUtility::registerPlugin()` to figure out the plugin
+    signiture to use:
```

Comment:
```suggestion
    signature to use:
```

Reply:
You are right, I planned to do that, then got distracted and forgot.